sci-electronics/librepcb/librepcb-9999.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
Add version snapshot as well

Reply:
I don't know what is this and I can't see any difference [here](https://devmanual.gentoo.org/ebuild-writing/file-format/index.html) (#ebuild header).

Reply:
I mean nobody is going to add only live ebuild to the main tree, it is useless

Reply:
Oh, there isn't such a release yet. That's why I wrote that adding it might help the developers. It's ok, anyone who might find it here could use it. Untill a stable release, the ebuild still works for me :)

Reply:
then you have to take a snapshot of master for ebuild

Reply:
Self-detected version 0.1.0 (first_pcb-539-g5813573-dirty). Seems version 0.1.0_p201805xx will useful... Is it OK?

Reply:
I will see about this the following days, when I have time. Thanks :)

Reply:
ping :)

Reply:
I am not sure `1122a91` didn't get verified though.